homeassistant/components/music_assistant/entity.py

```diff
@@ -0,0 +1,91 @@
+"""Base entity model."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from music_assistant.common.models.enums import EventType
+from music_assistant.common.models.event import MassEvent
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+if TYPE_CHECKING:
+    from music_assistant.client import MusicAssistantClient
+    from music_assistant.common.models.player import Player
+
+
+class MassBaseEntity(Entity):
+    """Base Entity from Music Assistant Player."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, mass: MusicAssistantClient, player_id: str) -> None:
+        """Initialize MediaPlayer entity."""
+        self.mass = mass
+        self.player_id = player_id
+        self._attr_should_poll = False
```

Comment:
Can be set outside of the constructor

Reply:
You mean the "_attr_should_poll" attribute?

Reply:
Yes